tasks/computer-vision/image-classification/ImageClassifier.py

```diff
@@ -0,0 +1,162 @@
+import pandas as pd
+import mlflow
+import mlflow.keras
+from keras.preprocessing.image import img_to_array, array_to_img
+import numpy as np 
+import os, time
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
+import keras
+from keras import models
+from keras.models import Model
+from keras import layers
+from keras import optimizers
+from keras import callbacks
+from keras.layers.advanced_activations import LeakyReLU
+from keras.utils import to_categorical
+from keras.models import Sequential
+from keras.layers import Conv2D, MaxPooling2D
+from keras.layers import Dense, Dropout, Flatten
+from keras.preprocessing.image import ImageDataGenerator
+from keras.applications import VGG16;
+from keras.applications.vgg16 import preprocess_input
+import os
+from sklearn.metrics import classification_report
+mlflow.keras.autolog()
```

Comment:
@varsha6319 Please add `mlflow>=1.1` to your `conda.yaml` file (since this is a [new feature](https://databricks.com/blog/2019/07/23/announcing-the-mlflow-1-1-release.html))